docs/develop/develop.md

```diff
@@ -25,6 +25,29 @@ You should make sure that everything in the UI looks green. If not, e.g. if the
 
 If everything is green, then you can already check for your clusterstack that has been deployed. You can use a tool like k9s to have a look at the management cluster and its custom resources. 
 
+Example:
+
+```shell
+❯ kubectl get clusterstacks -A
+NAMESPACE   NAME           PROVIDER   CLUSTERSTACK   K8S    CHANNEL   AUTOSUBSCRIBE   USABLE   LATEST                            AGE     REASON   MESSAGE
+cluster     clusterstack   docker     ferrol         1.27   stable    false           v2       docker-ferrol-1-27-v2 | v1.27.3   4m52s
+```
+
+```shell
+❯ kubectl get clusterstackreleases.clusterstack.x-k8s.io -A
+NAMESPACE   NAME                    K8S VERSION   READY   AGE     REASON   MESSAGE
+cluster     docker-ferrol-1-27-v2   v1.27.3       true    7m51s
+```
+
+The above cluster stack was downloaded from [SovereignCloudStack/cluster-stacks](https://github.com/SovereignCloudStack/cluster-stacks/releases)
+
+Up to now, no workload-cluster is in the management-cluster:
```

Comment:
from line 44 this is a bit unintuitive.. Can we remove this part or do you wanna re-write?

Reply:
@janiskemper I deleted that part. Of course the docs could get improved, but I would like to keep that PR small. Is it ready to merge?